bcda/main_test.go

```diff
@@ -219,13 +242,13 @@ func (s *MainTestSuite) TestCreateAlphaToken() {
 	// To execute all scenarios, invoke the rest of the tests directly (not by CLI)
 	// (this is required in order to validate the strings returned)
 
-	alphaTokenInfo, err := createAlphaToken("", "Dev")
+	alphaTokenInfo, err := createAlphaToken(0, "Dev")
```

Comment:
My intent with the signature on this method is to make the notion of "default TTL" belong to the CLI, not to token creation. We don't have any restrictions on the value of the ttl ... it can be negative at the moment. It is also the case that this is an expression in hours, so the minimum future expiration time we can set on ttl for a token atm is 1 hour.